contrib/namespace_cleanup.sh

```diff
@@ -5,14 +5,20 @@ ecosystem="stagef"
 read -p "Are you at tron-$ecosystem (y/n)?" RES
 echo
 if [ $RES = "y" ]; then
-        for file in /nail/tron/config/*
+        #load namespace from _manifest.yaml
+        for namespace in $(cat /nail/tron/config/_manifest.yaml | uq | jq -r 'keys[]')
         do
-                filename="$(basename $file)"
-                namespace="$(basename $file .yaml)"
+                file=$(cat /nail/tron/config/_manifest.yaml | uq | jq -r .\"$namespace\")
+                filename=$(basename $file)
                 if [ -f "/nail/etc/services/tron/$ecosystem/$filename" ]; then
                         echo "$namespace is up to date"
+                elif [ $namespace == "MASTER" ]; then
+                        echo "It is MASTER namepsace"
                 else
                         echo "$namespace is left behind, delete the namespace"
```

Comment:
Can we say "deleting the namespace"? This makes it seem like action is required from you.

Reply:
Sure.